templates/default/css/template.css

```diff
@@ -654,15 +655,24 @@ div.yes-no .btn-group label {
   .table-bordered{
     border:none;
   }
-
-  .ls-answers tr:last-child{
+  
+  .ls-answers tbody:last-child tr:last-child{
```

Comment:
There are multiple tbody statements in the the question (due to repeated headers?)